src/Components/Article/ArticleDescription.jsx

```diff
@@ -37,13 +42,28 @@ function ArticleDescription() {
                     </p>
                     <p>
                         <b>Votes : </b>
-                        {article.votes}
+                        {votes}
                     </p>
                     <p>
                         <b>Comments made :</b> {article.total_comments}
                     </p>
+                    <Vote
+                        article={article}
+                        votes={votes}
+                        setVotes={setVotes}
+                        setArticle={setArticle}
+                    />
                 </div>
             </div>
+
+            <h3
+                className={styles.comment}
+                onClick={() => setViewComments((prevState) => !prevState)}
+            >
+                Click here to view comments
+            </h3>
+
```

Comment:
this button always says "Click here to view comments" even when clicking it would *hide* the comments

You can conditionally render text in React too!

```jsx
const isMorning = true

reuturn <p>{isMorning ? "good morning!" : "good afternoon!"}</p>

```